frigate/comms/dispatcher.py

```diff
@@ -77,18 +60,25 @@ def __init__(
             "motion": self._on_motion_command,
             "motion_contour_area": self._on_motion_contour_area_command,
             "motion_threshold": self._on_motion_threshold_command,
+            "notifications": self._on_camera_notification_command,
             "recordings": self._on_recordings_command,
             "snapshots": self._on_snapshots_command,
             "birdseye": self._on_birdseye_command,
             "birdseye_mode": self._on_birdseye_mode_command,
         }
         self._global_settings_handlers: dict[str, Callable] = {
-            "notifications": self._on_notification_command,
+            "notifications": self._on_global_notification_command,
         }
 
         for comm in self.comms:
             comm.subscribe(self._receive)
 
+        self.web_push_client = next(
+            (comm for comm in communicators if isinstance(comm, WebPushClient)), None
+        )
+        if self.web_push_client is None:
+            logger.warning("WebPushClient not found in communicators")
```

Comment:
Won't this log if a user doesn't have notifications enabled for any camera?

Reply:
yep, thanks for catching that, fixed